concepticondata/conceptlists/Miedema-1985-40.tsv

```diff
@@ -0,0 +1,40 @@
+Miedema & Welling-1985-41-1	1	arm	1673	ARM
```

Comment:
Ah, sorry, there are missing headers:

```text
ID   NUMBER   ENGLISH   CONCEPTICON_ID   CONCEPTICON_GLOSS
```

Note also that our ID system has the convention to only take the first author, so it should be Miedema-1985-40-X in the ID field, as this needs to be identical with the name of the concept list.